distsql/stream.go

```diff
@@ -147,6 +152,10 @@ func (r *streamResult) NextRaw(goCtx goctx.Context) ([]byte, error) {
 }
```

Comment:
Should we also update the `partialCount` and `scanKeys`?

Reply:
We can't get scanKeys for the raw response.

Reply:
I mean we can set the `scanKeys` to -1 like the `selectResult` do.